Framework/API/inc/MantidAPI/IndexProperty.h

```diff
@@ -50,17 +53,20 @@ class MANTID_API_DLL IndexProperty : public Kernel::ArrayProperty<int> {
 
   IndexProperty *clone() const override;
 
-  using Kernel::ArrayProperty<int>::operator=;
+  using Kernel::ArrayProperty<int64_t>::operator=;
 
   bool isDefault() const override;
   std::string isValid() const override;
   std::string operator=(const std::string &rhs);
   operator Indexing::SpectrumIndexSet() const;
   Indexing::SpectrumIndexSet getIndices() const;
+  Indexing::IndexInfo getFilteredIndexInfo() const;
```

Comment:
Neither in declaration or implementation is there a description on what you or filtering on or how it works.